src/gt4py/backend/base_gt_backend.py

```diff
@@ -498,15 +498,13 @@ def validate_cache_info(cls, stencil_id, cache_info):
 
     @classmethod
     def build(cls, stencil_id, performance_ir, definition_func, options):
+        from gt4py import gt_src_manager
+
         cls._check_options(options)
 
         # Generate the Python binary extension (checking if GridTools sources are installed)
-        if not os.path.isfile(
-            os.path.join(gt_config.GT_INCLUDE_PATH, "gridtools", "common", "defs.hpp")
-        ):
-            raise RuntimeError(
-                "Missing GridTools sources. Run 'python setup.py install_gt_sources'."
-            )
+        if not gt_src_manager.has_gt_sources() and not gt_src_manager.install_gt_sources():
+            raise RuntimeError("Missing GridTools sources.")
```

Comment:
Why remove the hint `Run 'python -m gt4py.gt_src_manager install'` ?

Reply:
If the code gets to this point, the hint should have been already printed in the failed `gt_src_manager.install_gt_sources` function